scripts/cross/Dockerfile

```diff
@@ -12,12 +12,16 @@ RUN apt-get update -y && apt-get install --no-install-recommends -y -q \
                          libltdl-dev \
                          libltdl7
 
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get install -y nodejs npm
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+
```

Comment:
oh wow are we totally abandoning npm?

Reply:
Currently it's only used to rebuild the C bindings for libsass - it should get bundled with the NodeSource install though (the explicit npm install there was just added recently). We also used to npm install yarn, but we're doing that from aptitude now as well.